CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## 1.5.0 - TODO: To be released
+## 1.6.0 - To be released
```

Comment:
not a blocker or smth, but would be changed in future: I will re-check, looks like it was not backmerged by me to develop after 1.5.0 release